components/kubeflow-notebooks/kustomize/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jupyter-notebook
+  labels:
+    app: jupyter-notebook
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jupyter-notebook
+  template:
+    metadata:
+      labels:
+        app: jupyter-notebook
+    spec:
+      serviceAccountName: fl-notebook
+      containers:
+        - env:
+          - name: NB_PREFIX
+            value: /jupyter
+          - name: KF_PIPELINES_SA_TOKEN_PATH
+            value: /var/run/secrets/kubeflow/pipelines/token
+          # TODO: Parametrize
+          image: localhost:5000/katulu-fl-jupyter-full:1657174761
```

Comment:
This needs to be parameterized as the TODO said before we can merge this to install the image from the correct registry.

Reply:
Thanks for the reminder. Removed!